apps/ui/src/components/MessageVotingPower.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import { VotingPowerItem } from '@/stores/votingPowers';
+
+const props = defineProps<{
+  minVotingPower?: bigint;
+  minProposalThreshold?: bigint;
+  votingPower: VotingPowerItem;
+}>();
+
+defineEmits<{
+  (e: 'fetchVotingPower');
+}>();
+
+const insufficientVoteVp = computed(
+  () =>
+    props.votingPower.status === 'success' &&
+    props.minVotingPower !== undefined &&
+    props.votingPower.totalVotingPower <= props.minVotingPower
+);
+
+const insufficientProposeVp = computed(
+  () =>
+    props.votingPower.status === 'success' &&
+    props.minProposalThreshold !== undefined &&
+    props.votingPower.totalVotingPower < props.minProposalThreshold
+);
```

Comment:
This doubles what we are already computing in `useVotingPower` composable. Would be good to centralise it.

2 out of 3 places we could just access it directly from the composable, in one case (ModalVotingPower, we will need to pass it through the parent)

Reply:
Now save as `canPropose` and `canVote` property directly on the VotingPower object.